subprojects/gradle/ci-logger/src/main/kotlin/com/avito/utils/logging/Gradle.kt

```diff
@@ -120,6 +120,14 @@ private fun defaultCILogger(
     )
 }
 
+private fun elasticClient(project: Project): ElasticClient {
```

Comment:
@dsvoronin, is it ok to create multiple sentry clients in different task actions?

Reply:
It is similar to creating multiple retrofit(and so okhttp) clients